### PR TITLE
feat(pattern): quote pattern created

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Related Ticket(s)

#1859 

### Description

The Quote pattern was created using the design and functional specs

Screen Shot 2020-04-30 at 4 20 25 PM

### CODE REVIEW

File: packages/react/src/patterns/sub-patterns/Quote/README.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+> 💡 Don't forget to import the quote styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+## Props
+
+| Name      | Required | Data Type | Default Value  | Description                                                                                                                                             |
+| --------- | -------- | --------- | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `copy`    | YES      | String    | null           | Main Quote                                                                                                                                              |
+| `types`   | NO       | String    | `doubleCurved` | See `Types` bellow.                                                                                                                                     |
```
Comment (jeffchew) [inline packages/react/src/patterns/sub-patterns/Quote/README.md:62]: minor thing, should it be `type` and not `types`?

Comment (RaphaelAmadeu) [reply packages/react/src/patterns/sub-patterns/Quote/README.md:62]: i've fixed that on the latest commit, good catch

File: packages/react/src/patterns/sub-patterns/Quote/__stories__/Quote.stories.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+storiesOf('Patterns (Sub-Patterns)|Quote', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const copy = text(
+      'Quote(Required): ',
```
Comment (jeffchew) [inline packages/react/src/patterns/sub-patterns/Quote/__stories__/Quote.stories.js:24]: a format I started following is to include the prop name in parenthesis, might be worth following (Carbon seems to do this too now):

```
const copy = text('quote text (copy)', 'Lorem Ipsum');
const markType = select('quotation mark type (markType):', types, types.doubleCurved);
...etc...
```

Comment (RaphaelAmadeu) [reply packages/react/src/patterns/sub-patterns/Quote/__stories__/Quote.stories.js]: I've updated it on the latest commit

File: packages/react/src/patterns/sub-patterns/Quote/Quote.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+const { prefix } = settings;
+
+/**
+ * Quote Pattern
+ *
+ * @param {object} props props Object
+ * @param {string} props.markType type of the quote marks
+ * @param {string} props.copy quote copy
+ * @param {object} props.source source object
+ * @param {object} props.source.heading name heading for quote source
```
Comment (jeffchew) [inline packages/react/src/patterns/sub-patterns/Quote/Quote.js:29]: @RaphaelAmadeu can you check the types here? I don't think all are objects...

Comment (RaphaelAmadeu) [reply packages/react/src/patterns/sub-patterns/Quote/Quote.js]: @jeffchew I've updated this in the latest commit, Thanks!

**Pull request merged**